### PR TITLE
YearMonth converters

### PR DESCRIPTION
**Pull request by Wallman**

### CODE REVIEW

File: ymer/src/test/java/com/avanza/ymer/support/JavaYearMonthConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+public class JavaYearMonthConverterTest {
+
+    private JavaYearMonthReadConverter javaYearMonthReadConverter;
+    private JavaYearMonthWriteConverter javaYearMonthWriteConverter;
+
+    @Before
+    public void test() {
```
Comment (ath0s) [inline ymer/src/test/java/com/avanza/ymer/support/JavaYearMonthConverterTest.java:36]:
```suggestion
    public void setup() {
```
Maybe this method should have a different name? An alternative would be to just initialize the fields in their declaration, and remove this method.

File: ymer/src/test/java/com/avanza/ymer/support/JavaYearMonthConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        javaYearMonthReadConverter = new JavaYearMonthReadConverter();
+        javaYearMonthWriteConverter = new JavaYearMonthWriteConverter();
+    }
```
Comment (ath0s) [inline ymer/src/test/java/com/avanza/ymer/support/JavaYearMonthConverterTest.java:39]:
```suggestion
    private JavaYearMonthReadConverter javaYearMonthReadConverter = new JavaYearMonthReadConverter();
    private JavaYearMonthWriteConverter javaYearMonthWriteConverter = new JavaYearMonthWriteConverter();    
```

File: ymer/src/test/java/com/avanza/ymer/support/JavaYearMonthReadConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import org.junit.Test;
+
+public class JavaYearMonthReadConverterTest {
+
+	private JavaYearMonthReadConverter javaYearMonthReadConverter;
+
+	@Before
+	public void setup() {
+		javaYearMonthReadConverter = new JavaYearMonthReadConverter();
+	}
```
Comment (ath0s) [inline ymer/src/test/java/com/avanza/ymer/support/JavaYearMonthReadConverterTest.java:33]:
```suggestion
	private JavaYearMonthReadConverter javaYearMonthReadConverter = new JavaYearMonthReadConverter();
```

File: ymer/src/test/java/com/avanza/ymer/support/JavaYearMonthWriteConverterTest.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import org.junit.Test;
+
+public class JavaYearMonthWriteConverterTest {
+
+	private JavaYearMonthWriteConverter javaYearMonthWriteConverter;
+
+	@Before
+	public void setup() {
+		javaYearMonthWriteConverter = new JavaYearMonthWriteConverter();
+	}
```
Comment (ath0s) [inline ymer/src/test/java/com/avanza/ymer/support/JavaYearMonthWriteConverterTest.java:36]:
```suggestion
	private JavaYearMonthWriteConverter javaYearMonthWriteConverter;
 = new JavaYearMonthWriteConverter();
```

**Review by ath0s — APPROVED** [review]

**Review by kbs4j — COMMENTED** [review]
Just a few small comments

**Review by kbs4j — APPROVED** [review]

**Pull request merged**